pkg/version/root.go

```diff
@@ -68,3 +69,16 @@ func (c *RootCommand) Exec(in io.Reader, out io.Writer) error {
 	fmt.Fprintf(out, "Built with %s\n", GoVersion)
 	return nil
 }
+
+// PreRelease determines if the given app version is a pre-release.
+//
+// NOTE: this is indicated by the presence of a hyphen, e.g. v1.0.0-rc.1
+func PreRelease(version string) bool {
```

Comment:
Personal style, but I normally prefix predicate functions like this with `IsFoo` so the conditional reads nicer.

```suggestion
func IsPreRelease(version string) bool {
```